torch/nn/modules/module.py

```diff
@@ -273,6 +273,9 @@ def cpu(self):
         """
         return self._apply(lambda t: t.cpu())
 
+    def to_mkldnn(self):
```

Comment:
No, we can't do that!!!! We need a unified interface for all backends!! This is the main subclass for half of our library, don't put random functions in there!